ddcz/models/used/discussion.py

```diff
@@ -29,3 +29,22 @@ def user_profile_url(self):
     @property
     def by_registered_user(self):
         return self.reg == "1" and self.user
+
+    def createNewCommentFromForm(self, form, user, reputation=0):
+        self.nickname = user.nick_uzivatele
+        self.email = user.email_uzivatele
+        self.text = form.cleaned_data["text"]
+        self.reg = 1
+        self.reputace = reputation
+        self.user = user
+        self.save()
+
+    def deleteComment(self, comment_id):
```

Comment:
It is weird to reference to  different object...

Reply:
deleted (now that's a pun :D )